### PR TITLE
before editorconfig

### PR DESCRIPTION
**Pull request by kwacky1**
This is my default PR Template

### CODE REVIEW

**Review by lesleyanneb — COMMENTED** [review]
Are you sure this will be good enough?

File: bin/Debug/netcoreapp3.1/hellocsharp.deps.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "name": ".NETCoreApp,Version=v3.1",
     "signature": ""
   },
-  "compilationOptions": {},
+  "compilationOptions": {making a change for changes sake},
```
Comment (lesleyanneb) [inline bin/Debug/netcoreapp3.1/hellocsharp.deps.json]:
```suggestion
  "compilationOptions": {making a change for the future},
```

**Review by lesleyanneb — CHANGES_REQUESTED** [review]
I think this will be better

**Comment by kwacky1** [comment]
Thanks @lesleyanneb

**Review by lesleyanneb — APPROVED** [review]

**Comment by lesleyanneb** [comment]
Can we close this PR or merge it

**Pull request merged**